src/Entity/Field.php

```diff
@@ -80,7 +80,7 @@ class Field implements FieldInterface, TranslatableInterface
     private $version;
 
     /**
-     * @ORM\ManyToOne(targetEntity="Bolt\Entity\Content", inversedBy="fields", fetch="EAGER")
+     * @ORM\ManyToOne(targetEntity="Bolt\Entity\Content", inversedBy="fields")
```

Comment:
The WITH query should be adjusted instead.

Reply:
Changed.